quote-generator/src/app/app.component.html

```diff
@@ -3,12 +3,20 @@
     <span>random</span>
     <img src="../assets/round-arrows-20x20.png" />
   </button>
-  <main *ngIf="quote">
-    <div class="quote-text">{{ quote.text }}</div>
-    <div class="quote-author">{{ quote.author }}</div>
-    <div class="quote-genre">{{ quote.genre }}</div>
+  <main>
+
+    <quote-random
+      [quote]="quote"
```

Comment:
Jakiś to sposób jest za ngIfować, ale jak chcielibyśmy już mieć projekt do perfekcji musisz to zmienić na , a quote-random i quote-list przenieść do oddzielnych modułów